docs/ref/validators.txt

```diff
@@ -161,14 +161,22 @@ to, or in lieu of custom ``field.clean()`` methods.
 
     .. attribute:: schemes
 
+        .. versionchanged:: 1.10
+
         URL/URI scheme list to validate against. If not provided, the
-        default is ``['http', 'https', 'ftp', 'ftps']``. If the special
-        valid ``__all__`` is passed, then all schemas are allowed.
+        default is ``['http', 'https', 'ftp', 'ftps']``.
+
+        Schemes must start with a letter, and may optionally contain
+        an addition protocol part appended with a `+`, ie. `git+ssh`.
```

Comment:
ie -> e.g.
must double the backticks for them to have an effect on the rendered docs